src/components/Web3Status/index.tsx

```diff
@@ -174,15 +153,6 @@ function Web3StatusInner() {
 
   if (!chainId) {
     return null
-  } else if (error) {
-    return (
-      <Web3StatusError onClick={handleWalletDropdownClick}>
-        <NetworkIcon />
-        <Text>
-          <Trans>Error</Trans>
-        </Text>
-      </Web3StatusError>
-    )
```

Comment:
Is there any more generic error (not user rejection) that we do want to preserve this error for? (I'm thinking like the wallet genuinely failed to switch...?)

Reply:
yeah in the linear ticket it actually said to remove that for this and just show a toast